tools/cldr/src/com/ibm/icu/dev/tool/cldr/IcuConverterConfig.java

```diff
@@ -0,0 +1,212 @@
+package com.ibm.icu.dev.tool.cldr;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Optional;
+import java.util.Set;
+import java.util.function.Predicate;
+
+import org.unicode.cldr.api.CldrDraftStatus;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * The converter config intended to generate the standard ICU data files.
+ */
+// TODO: Bind this to Ant or some other more user friendly config file format.
+// TODO: Uncomment "ars" once that's in CLDR.
+final class IcuConverterConfig implements LdmlConverter.ConverterConfig {
+    // The set of locale IDs used by ICU. Some of these are aliases however, so don't expect
+    // to find XML files for all of them.
+    private static final ImmutableSet<String> ICU_LOCALE_IDS = ImmutableSet.of(
```

Comment:
Copied from the ICU Ant config file as of release 35.1 (same for everything else here).